prov/efa/src/efa_domain.c

```diff
@@ -301,11 +303,11 @@ int efa_domain_open(struct fid_fabric *fabric_fid, struct fi_info *info,
 			                         EFA_MR_CACHE_LIMIT_MULT;
 		cache_params.max_cnt = efa_mr_max_cached_count;
 		cache_params.max_size = efa_mr_max_cached_size;
-		domain->cache.entry_data_size = sizeof(struct efa_mr);
-		domain->cache.add_region = efa_mr_cache_entry_reg;
-		domain->cache.delete_region = efa_mr_cache_entry_dereg;
+		domain->cache->entry_data_size = sizeof(struct efa_mr);
```

Comment:
This commit is not correct; cache has no backing when you write into it.  I think you fixed this in a later commit, but we should strive to have all commits functional, to not break bisectability.

Reply:
Ah yeah my bad, my hunks weren't splittable so I manually deleted and re-added code, must have messed up here.